src/inventory/dispense/dispense-page.jsx

```diff
@@ -123,6 +126,13 @@ export const DispensePage = () => {
 									</TableRow>
 								</TableHead>
 								<TableBody>
+								{rows.length === 0 && (
+									<TableRow>
+										<div style={{fontSize: "20px"}}>
+										Currently there are no active drug orders
```

Comment:
Message displayed is not correct. change it to something like 'No active patients with drug orders'

Reply:
Done.